Pytest fails on tests for apps/sections/tests.py

## Type
Bug Report

## Problem or To-Do
I'm getting errors on `test_sections_query_without_data` and `test_sections_query_with_data`. Both throw and error similar to

```bash
E       TypeError: object of type 'NoneType' has no len()
apps/sections/tests.py:91: TypeError
```

@Ryan104 originally was kind enough to write these tests. Please take a look and troubleshoot!

--- a/apps/sections/schema.py
+++ b/apps/sections/schema.py
@@ -54,7 +54,7 @@ class CreateSection(graphene.Mutation):
 
         # Return all parameters
         return CreateSection(
-            id=bed.id,
+            id=section.id,
             end_date=end_date,
             is_active=is_active,
             start_date=start_date,
@@ -67,7 +67,7 @@ class CreateSection(graphene.Mutation):
 class Query(graphene.ObjectType):
     sections = graphene.List(SectionType)
 
-    def resolve_all_sections(self, info):
+    def resolve_sections(self, info):
         return Section.objects.all()
 
 

--- a/apps/sections/tests.py
+++ b/apps/sections/tests.py
@@ -78,17 +78,16 @@ class SectionQueryTests(GraphQLTestCase):
     def test_sections_query_without_data(self):
         res = self.query(self.SECTION_QUERY)
         data = json.loads(res.content)['data']
-        assert data['sections'] == None
+        assert len(data['sections']) == 0
 
-    # def test_sections_query_with_data(self):
-    #     section = _create_default_section()
-    #     res = self.query(self.SECTION_QUERY)
-    #     data = json.loads(res.content)['data']
-    #     print(data)
+    def test_sections_query_with_data(self):
+        section = _create_default_section()
+        res = self.query(self.SECTION_QUERY)
+        data = json.loads(res.content)['data']
 
-    #     assert len(data['sections']) == 1
-    #     assert data['sections'][0]['id'] == str(section.id)
-    #     assert data['sections'][0]['bed']['name'] == 'Secret Bed'
+        assert len(data['sections']) == 1
+        assert data['sections'][0]['id'] == str(section.id)
+        assert data['sections'][0]['bed']['name'] == 'Secret Bed'
 
 
 class SectionMutationTest(GraphQLTestCase):

